model/registry/cloudfoundry.yaml

```diff
@@ -0,0 +1,158 @@
+groups:
+  - id: registry.cloudfoundry
+    prefix: cloudfoundry
```

Comment:
Prefix is not allowed anymore. Please add the value here in all attributes. Like `cloudfoundry.system.id`. This should fix the policy errors

> Group 'cloudfoundry.process' uses prefix 'cloudfoundry.process'. All attribute should be fully qualified with their id, prefix is no longer supported.

Reply:
Thanks, will do.